tests/data/cache/astropy/download/url/e9b5d99b7f6684fbea4a485dc468b804/url

```diff
@@ -0,0 +1 @@
+https://archive.stsci.edu/hlsps/reference-atlases/cdbs/calspec/hd111980_stis_005.fits
```

Comment:
The fix is to permanently cache the file and so never go to the internet?

Reply:
Yup!  So the tests can run happily no matter what.